modules/verilator/5.034/presubmit.yml

```diff
@@ -0,0 +1,12 @@
+bcr_test_module:
+  module_path: ""
+  matrix:
+    platform: ["ubuntu2004", "ubuntu2004_arm64", "macos_arm64"]
```

Comment:
Is windows supported?

Reply:
Windows is but I don't have a windows machine to develop it on and don't feel like going through the headache of developing it using CI. I'll give it one go but otherwise it's not something I was gonna implement initially.

Reply:
Ah lol I just opened https://github.com/jmillikin/rules_flex/issues/14